Tiltfile

```diff
@@ -176,8 +177,8 @@ for resource in manager_yaml:
         resource["stringData"]["dnsToken"] = os.getenv("LINODE_DNS_TOKEN")
         if os.getenv("LINODE_URL"):
             resource["stringData"]["LINODE_URL"] = os.getenv("LINODE_URL")
-        if os.getenv("LINODE_CA_BASE64"):
-            resource["stringData"]["SSL_CERT_DIR"] = "/tmp/linode-ca"
+    if resource["metadata"]["name"] == "capl-ca":
+        resource["data"]["cacert.pem"] = os.getenv("LINODE_CA_BASE64")
```

Comment:
Since its already base64 encoded, it should be stringData here. Adding full cert content as env var was causing issues, so it was base64 encoded and we added the var LINODE_CA_BASE64